docs/antrea-ipam.md

```diff
@@ -221,7 +221,8 @@ where the underlay router will route the traffic to the destination VLAN.
 ### Requirements for this Feature
 
 As of now, this feature is supported on Linux Nodes, with IPv4, `system` OVS datapath
-type, and `noEncap`, `noSNAT` traffic mode.
+type, `noEncap`, `noSNAT` traffic mode, and `AntreaProxy` feature enabled.
+Working with `ProxyAll` feature is not verified.
```

Comment:
Working -> Configuration

Nit: no need to add a new line if it is the same paragraph.

Reply:
Thanks, fixed.